src/react-components/room/object-hooks.js

```diff
@@ -0,0 +1,145 @@
+import { useEffect, useState, useCallback, useMemo } from "react";
+import { removeNetworkedObject } from "../../utils/removeNetworkedObject";
+import { rotateInPlaceAroundWorldUp, affixToWorldUp } from "../../utils/three-utils";
+import { getPromotionTokenForFile } from "../../utils/media-utils";
+
+function getPinnedState(el) {
+  return el.components.pinnable && el.components.pinnable.data.pinned;
+}
+
+function hasIsStaticTag(el) {
+  return el.components.tags && el.components.tags.data.isStatic;
+}
+
+export function getPlayerInfo(object) {
+  return object.el.components["player-info"] && object.el.components["player-info"].data;
+}
```

Comment:
This is only used to determine if an object is a player. Maybe just call it `isPlayer` and have it strictly return a boolean.